spec/features/admin_creates_a_product_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Admin creates a proudct" do
```

Comment:
You describe things as "Admin" but currently I don't see anything that would indicate there is an admin role. Is that something that you have another story for? If that functionality doesn't exist yet it might be good to name these feature specs in such a way to indicate it isn't actually implemented that way just yet. If the admin functionality is suppose to be there and it just got missed then cool. Or if I am missing something just let me know.